File: backend/src/main/java/cohappy/backend/model/dto/request/CreateHouseAdvertisementDTO.java
package cohappy.backend.model.dto.request;

import lombok.Data;

import java.util.List;

@Data
public class CreateHouseAdvertisementDTO {
    private List<byte[]> images;
    private String houseCode;
    private HouseStateDTO state;
    private String publishedBy;
    private String description;
}

File: frontend/app/src/main/java/cohappy/frontend/model/dto/request/CreateHouseAdvertisementDTO.kt
package cohappy.frontend.model.dto.request

import cohappy.frontend.model.dto.HouseStateEnum
import com.squareup.moshi.JsonClass

@JsonClass(generateAdapter = true)
data class CreateHouseAdvertisementDTO(
    val houseCode: String,
    val state: HouseStateEnum,
    val publishedBy: String,
    var description: String? = null
)
